### DISCUSSION TITLE
有时候启动服务器会报错，bufferedBytes has overflowed(2068 > 2048)

### DISCUSSION THREAD

**Discussion by MacroGu** [kbengine/kbengine#1372] [Q&A]
具体报错日志：
` baseapp-1: [ERROR]: Channel::onPacketReceived[0x1e14b122480]: external channel(192.168.31.10:60985/1/0/0), bufferedBytes has overflowed(2156 > 2048), Try adjusting the kbengine[_defs].xml->windowOverflow->receive.
   baseapp-1: [ERROR]: Channel::processPackets(192.168.31.10:60985/1/16/0): channel[0x1e14b122480] is condemn.
   baseapp-1: [ERROR]: Channel::onPacketReceived[0x1e14b1724e0]: external channel(192.168.31.10:60985/2/0/0), bufferedBytes has overflowed(2068 > 2048), Try adjusting the kbengine[_defs].xml->windowOverflow->receive.
   baseapp-1: [ERROR]: Channel::processPackets(192.168.31.10:60985/2/16/0): channel[0x1e14b1724e0] is condemn.`

但是再次重启又好了， 请问什么原因，是不是有什么bug， 不是必现的错误log

**Comment by kebiao** [comment]
external channel(192.168.31.10:60985/2/0/0), bufferedBytes has overflowed


提示多看看就清楚了

**Comment by kebiao** [comment]
客户端发包超过限制了

**Reply by MacroGu** [reply]
那应该是我重启服务器后，我客户端没有关闭一直在发送所导致的
